### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by Liliya-V**
Created an admin panel, CSSed it

### CODE REVIEW

File: adminPanel/adminPanel.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        echo '<br><br>';
+        echo '<table border=5>
+        <tr>
+            <th>TITLE</th>
+            <th colspan="2">ACTION</th>
+        </tr>';
+        foreach ($titles as $title) {
+            echo '<tr>';
+            echo '<td>' .$title['title']. '</td>';
+            echo '<td><a href="#">Edit</a></td>';
```
Comment (snack-attack) [inline adminPanel/adminPanel.php]: you could combine these echos - you don't necessarily need to write an echo statement for every line of html but it is easier to read this way I suppose

File: adminPanel/adminPanel.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="../css/styleAdminPanel.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+ <body>
+    <h1>Projects management panel</h1>
+      <?php
+        echo '<td><a href="#">Add new projet</a></td>';
+        echo '<br><br>';
```
Comment (snack-attack) [inline adminPanel/adminPanel.php]: avoid using <br> tags for styling. Preference adding a class and margin-bottom in css

Comment (Liliya-V) [reply adminPanel/adminPanel.php]: got it

File: adminPanel/adminPanel.php (hunk 1 above)
Comment (Liliya-V) [reply adminPanel/adminPanel.php]: yes i do find it easier to read

File: adminPanel/adminPanel.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <title>Portfolio Liliya Voevodina</title>
+        <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="../css/styleAdminPanel.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+ <body>
+    <h1>Projects management panel</h1>
+      <?php
+        echo '<h3 class="addbutton"><a href="#">Add new projet</a></h3>';
```
Comment (sjr-mayden) [inline adminPanel/adminPanel.php]: Project is missing a 'c'

File: css/styleAdminPanel.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+*{
+    box-sizing:border-box;
+}
+body {
```
Comment (sjr-mayden) [inline css/styleAdminPanel.css:5]: Missing a line of whitespace here

File: adminPanel/adminPanel.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=PortfolioProjects", 'root', '');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```
Comment (sjr-mayden) [inline adminPanel/adminPanel.php]: Extra spaces around the method accessor, should be $db->setAttr

File: adminPanel/adminPanel.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+$sql="SELECT `title` FROM `projects`;";
```
Comment (sjr-mayden) [inline adminPanel/adminPanel.php]: Missing spaces around assignation, should be $sql = "

**Review by sjr-mayden — CHANGES_REQUESTED** [review]
Suggest changing from absolute to relative path on image field in database

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**